src/main/java/edu/cornell/kfs/fp/batch/service/impl/AccountingDocumentGeneratorBase.java

```diff
@@ -66,7 +66,7 @@ public T createDocument(Function<Class<? extends Document>, Document> emptyDocum
     }
 
     protected void populateDocumentHeader(T document, AccountingXmlDocumentEntry documentEntry) {
-        FinancialSystemDocumentHeader documentHeader = (FinancialSystemDocumentHeader) document.getDocumentHeader();
+        DocumentHeader documentHeader = (DocumentHeader) document.getDocumentHeader();

```

Comment:
The DocumentHeader typecast is no longer needed.

Reply:
updated